### ISSUE TITLE
[FEATURE] Google Translate Command

### ISSUE THREAD

**Issue by rosalogia** [rosalogia/rucs24-bot#64]
### Specify what type of functionality your request pertains to

- [x] Command
- [ ] Other

### Describe how it should work

Command to translate some text to a specified language, English by default

#### [Optionally] Describe a potential solution

This [Google Translate Python Library](https://pypi.org/project/googletrans/) is probably more than enough to implement this feature

**Comment by PAndaContron** [comment]
Alternatively, Google Translate also has a hidden API that I've used for this in the past ([full code here](https://repl.it/@PAndaContron/The-Translation-Paradox-Demonstration#main.py)).

You can use the API just by sending a request to `https://translate.googleapis.com/translate_a/single?client=gtx` using the URL encoded parameters `sl` for source language, `tl` for translation language, `dt=t`, and `q` for the string you want to translate.

Instead of implementing that yourself, you can actually just copy this translate function from the code above directly, since it's completely separate from the rest of that program's logic:

```py
def translate(text, fromLang, toLang):
  url = "https://translate.googleapis.com/translate_a/single?client=gtx&sl=" + fromLang + "&tl=" + toLang + "&dt=t&q=" + urllib.parse.quote(text, safe='')
  r = requests.get(url = url)
  return r.json()[0][0][0]
```

For automatic language detection, set `fromLang` to `"auto"`. For other languages, both `fromLang` and `toLang` use the ISO codes from [this table](https://cloud.google.com/translate/docs/languages). I've already put all of these languages into an array in the code linked above, but that might be slightly out of date since it was almost a year ago, and you'd probably want to match them to their corresponding language names (i.e. `"English"` → `"en"`) for the bot.

Note that there is a rate limit, although it's very lenient; I only hit it when I was sending hundreds of requests a second, which isn't going to happen for any reasonable use case. For normal use cases, this API should be sufficient. It also doesn't require adding another library, which is probably an advantage over the Python library. Also, according to the Python library's PyPI page, it could be blocked at any time, which is less likely to happen to this API as far as I'm aware.

**Comment by rosalogia** [comment]
Closed by #78
